### DISCUSSION TITLE
Mapping filter list name (UI) to .txt file on filesystem

### DISCUSSION THREAD

**Discussion by HachimanSec** [AdguardTeam/AdGuardHome#7087] [Q&A] [answered]
Hi everyone, 
I have a brief question about the filter list names. In the UI I see the various names. 
On the filesystem, in /opt/AdGuardHome/data/filters, I only see numbers.txt. 

Is there any way to see a mapping between names in the UI and the txt files?

Thx!

**Comment by ainar-g** [comment, marked as answer]
Just as a disclaimer, `data/` contains the data that is internal to AdGuard Home, so I wouldn't recommend building scripts that rely on anything about those directories and files.

With that said, you can see the IDs in the configuration file, in the `filters` array.

**Reply by HachimanSec** [reply]
Thank you @ainar-g this was the right hint.The `filters` array gives me the right information. 

Just to explain my use case. I am ingesting the log data from AdGuard into my SIEM and there I want to apply custom rules to give me certain alerts. 
For example a blocked request due to adtech is ignorable. A block because of a hit on a malicious domain by abuse.ch should be reported. 
And as the FilterListID is present in the JSON log string I needed to know which ID matches which list.
